--- bcg729-1.0.1/src/Makefile.am.orig	2015-11-02 15:06:33.000000000 +0100
+++ bcg729-1.0.1/src/Makefile.am	2015-11-10 22:24:22.267248581 +0100
@@ -1,8 +1,4 @@
-if BUILD_MSBCG729
-noinst_LTLIBRARIES = libbcg729.la
-else
 lib_LTLIBRARIES = libbcg729.la
-endif
 
 libbcg729_la_SOURCES=	LP2LSPConversion.c \
 			LPSynthesisFilter.c \
--- bcg729-1.0.1/include/bcg729/Makefile.am.orig	2015-11-02 15:06:33.000000000 +0100
+++ bcg729-1.0.1/include/bcg729/Makefile.am	2015-11-10 22:24:47.163914203 +0100
@@ -3,8 +3,6 @@
 public_headers=encoder.h decoder.h
 
 bcg729_include_HEADERS=
-if !BUILD_MSBCG729
 bcg729_include_HEADERS+=$(public_headers)
-endif
 
 EXTRA_DIST=$(public_headers)
