diff -urNp -x '*.orig' bcg729-1.0.2.org/include/bcg729/Makefile.am bcg729-1.0.2/include/bcg729/Makefile.am
--- bcg729-1.0.2.org/include/bcg729/Makefile.am	2016-08-11 11:23:50.000000000 +0200
+++ bcg729-1.0.2/include/bcg729/Makefile.am	2021-07-06 00:10:07.382523842 +0200
@@ -3,7 +3,5 @@ bcg729_includedir=$(includedir)/bcg729
 public_headers=encoder.h decoder.h
 
 bcg729_include_HEADERS=
-if !BUILD_MSBCG729
 bcg729_include_HEADERS+=$(public_headers)
-endif
 
